crates/sources/src/sources/uniswap_v3/graph_api.rs

```diff
@@ -102,7 +121,7 @@ pub struct RegisteredPools {
 #[derive(Debug, Deserialize, PartialEq)]
 #[serde(rename_all = "camelCase")]
 pub struct PoolData {
-    pub id: H160,
+    pub id: String,
```

Comment:
Yes, so the reason for this change is that the tick data have ids = `0x0001fcbba8eb491c3ccfeddc5a5caba1a98c4c28#-92200` which is `pool_address#tick_idx`, and there I saw that different data can have different types for ids.

Reply:
I saw that the id of a `Tick` looks different but that wouldn't require the type of `PoolData::id` to change as well, right?
Since you made the paginated query generic over the type to deserialize into, wouldn't it still do the right thing during deserialization? I believe the only thing which is required is to implement
```rust
impl ContainsId for PoolData {
    fn get_id(&self) -> String {
        self.id.clone()
    }
}
```
such that it correctly converts from `H160` to `String`.

I had hoped 👇 would enable that behavior:
```rust
#[derive(Debug, Deserialize)]
pub struct Data<T> {
    #[serde(alias = "pools", alias = "ticks")]
    pub inner: Vec<T>,
}
```

Reply:
Oh yes, I managed to understand differently the comment. Reverted, thanks.